src/dsp_tools/commands/excel2json/project.py

```diff
@@ -59,20 +60,22 @@ def excel2json(
 def _validate_folder_structure_get_filenames(data_model_files: str) -> tuple[list[Path], list[Path]]:
     if not Path(data_model_files).is_dir():
         raise UserError(f"ERROR: {data_model_files} is not a directory.")
-    folder = [x for x in Path(data_model_files).glob("*") if _non_hidden(x)]
-    processed_files = []
-    onto_folders, processed_onto = _get_and_validate_onto_folder(Path(data_model_files), folder)
-    processed_files.extend(processed_onto)
-    listfolder, processed_lists = _get_validate_list_folder(data_model_files, folder)
-    processed_files.extend(processed_lists)
-    if len(onto_folders) + len(listfolder) != len(folder):
+    sub_folders = [x for x in Path(data_model_files).glob("*") if _non_hidden(x) and x.is_dir()]
+    files_to_process = []
+    onto_folder_content, onto_to_process = _get_and_validate_onto_folder(Path(data_model_files), sub_folders)
```

Comment:
question regarding onto_to_process: Are these the onto files to be processed? Then I suggest `onto_files_to_process`